.github/workflows/pr.yaml

```diff
@@ -0,0 +1,15 @@
+name: Publish Extension
+on: pull_request
+
+env:
+  ROAMJS_EXTENSION_ID: query-builder
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: install
+        run: npm install -f
+      - name: build
+        run: npx samepage build --review "ignore"
```

Comment:
this `ignore` is temporary. In the next version of `@samepage/scripts`, the `build` command will know not to push to Roam Depot review on PR branches automatically